exporter/awsxrayexporter/README.md

```diff
@@ -43,6 +43,7 @@ defined in the OpenTelemetry Semantic Conventions.
 | `aws.request_id` | AWS-generated unique identifier for the request.                       | No        |
 | `aws.queue_url`  | For operations on an Amazon SQS queue, the queue's URL.                | No        |
 | `aws.table_name` | For operations on a DynamoDB table, the name of the table.             | No        |
+| `aws.xray.annotations` | The attribute is a slice(list) attribute that contains each of the string keys. If found on the span, the awsxrayexporter will use them in addition to the indexed_attributes configuration field when categorizing which attributes to index.             | No        |
```

Comment:
nit:
```suggestion
| `aws.xray.annotations` | The attribute is a slice(list) attribute that contains each of the string keys. If found on the span, the `awsxrayexporter` will use them in addition to the `indexed_attributes` configuration field when categorizing which attributes to index.             | No        |
```

Reply:
Also, can you include an example to clarify what does the "slice" value of this attribute key would look like?